### ISSUE TITLE
Test failures relating to time zone

### ISSUE THREAD

**Issue by j0ni** [ihabunek/toot#227]
I'm packaging toot 0.28.1 for Void Linux. The packaging system attempts to run test suites when they exist, and there are 4 failures which I can't figure out.

They appear to be TZ related:

```
E           Zappa Old Fan @fan9@zappa-fans.social favourited your status
E         - Zappa Fan @fan123@zappa-fans.social                                             1983-11-04 16:53 CET
E         ?                                                                                             ^    -
E         + Zappa Fan @fan123@zappa-fans.social                                             1983-11-04 10:53 EST
E         ?                                                                                             ^     +
E
```

If I try running them with an explicit TZ, e.g., `env TZ=CET pytest -vv`, I see:

```
E           Zappa Old Fan @fan9@zappa-fans.social favourited your status
E         - Zappa Fan @fan123@zappa-fans.social                                             1983-11-04 16:53 CET
E         ?                                                                                             ^
E         + Zappa Fan @fan123@zappa-fans.social                                             1983-11-04 15:53 CET
E         ?   
```
So I'm not entirely sure how to get the TZ right. Is there some env I need to configure to make the tests behave well?

I don't see them in any of the CI config I've found in the repo, so maybe they're just not used any more. Some clarification would help here!

Thanks for an excellent little app. It's time has come :)

**Comment by ihabunek** [comment]
On Thu, 17 Nov 2022, at 23:42, Jon Irving wrote:
> I'm packaging toot 0.28.1 for Void Linux. The packaging system attempts to run test suites when they exist, and there are 4 failures which I can't figure out.

Seems like I made tests dependant on the timezone. I'll fix it for next release.

Regards,
Ivan

**Comment by ihabunek** [comment]
Probably next week.

**Comment by ihabunek** [comment]
@j0ni Fixed in [0.29.0](https://github.com/ihabunek/toot/releases/tag/0.29.0).
